lpa-submissions-e2e-tests/cypress/support/lpa-submission-tasklist/visitTaskListQuestionnaireSubmissionPage.js

```diff
@@ -0,0 +1,4 @@
+module.exports = () =>{
+    cy.visit('/lpa-submission/lpa-task-list')
+    cy.url().should('contain','/lpa-submission/lpa-task-list')
+}
```

Comment:
Add newline at end - use [editorconfig](https://editorconfig.org/) to do this for you